### PR TITLE
#5572 remove captureThrowable

### PR DESCRIPTION
**Pull request by davidradl**

### CODE REVIEW

**Comment by davidradl** [comment]
@lpalashevski tagging you to make you aware of this one; happy for any feedback.

**Review by lpalashevski — APPROVED** [review]
Looks good we should follow this pattern and update some of the omas-es. Noted, I will create technical debt item and follow up. Thanks for sharing...

**Comment by davidradl** [comment]
I will wait until the end of Friday and will merge if there is no untoward feedback.

**Pull request merged**
